tests/unit/lang/translator/providerFILE.cpp

```diff
@@ -0,0 +1,139 @@
+#include "catch2/catch.hpp"
+
+#include "translator.hpp"
+#include "translation_provider_FILE.hpp"
+#include "hash.hpp"
+#include "unaccent.hpp"
+#include <iostream>
+#include <iostream>
+#include <fstream>
+#include <cstdio>
+#include <deque>
+#include <set>
+#include <map>
+
+using namespace std;
+
+bool CompareStringView(string_view_utf8 s, string_view_utf8 s2, set<unichar> &nonAsciiChars, const char *langCode) {
```

Comment:
do we really need to copy this function from the other unit test?

Reply:
No, I made new header file with the common functions between the two providers